src/Nethermind/Nethermind.Blockchain.Test/GenesisBuilderTests.cs

```diff
@@ -2,16 +2,17 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 using System.IO;
+using FluentAssertions;
```

Comment:
Generally, I'd recommend avoiding a new code with FluentAssertions, as we're stuck with its old version.